pkg/client/unversioned/services.go

```diff
@@ -38,6 +38,7 @@ type ServiceInterface interface {
 	Delete(name string) error
 	Watch(label labels.Selector, field fields.Selector, opts api.ListOptions) (watch.Interface, error)
 	ProxyGet(scheme, name, port, path string, params map[string]string) ResponseWrapper
+	SubresourceProxyGet(scheme, name, port, path string, params map[string]string) ResponseWrapper
```

Comment:
when would I call this instead of ProxyGet? Why do we need two proxy functions? If the answer is "older servers won't support the subresource proxy", how is the caller supposed to know and decide which to call?